### PR TITLE
Rename XXXXQMLFrontEnd to XXXXQMLAdapter

### PR DESCRIPTION
**Pull request by rakeshsadhu**

### CODE REVIEW

File: src/desktop-dev-tools/ServiceMonitorQMLComponent.h [hunk 1]
```diff
@@ -61,9 +61,9 @@ class ServiceMonitorQMLComponent : public QObject
             if (m_target != nullptr) {
                 auto targetInterface = qobject_cast<InterfaceBase *>(m_target);
                 if (targetInterface == nullptr) {
-                    auto qmlFrontend = qobject_cast<QMLFrontendBase *>(m_target);
-                    if (qmlFrontend != nullptr) {
-                        targetInterface = qmlFrontend->provider();
+                    auto QMLAdapter = qobject_cast<QMLAdapterBase *>(m_target);
```
Comment (jacky309) [inline src/desktop-dev-tools/ServiceMonitorQMLComponent.h]: fix

Comment (rakeshsadhu) [reply src/desktop-dev-tools/ServiceMonitorQMLComponent.h]: done

**Comment by rakeshsadhu** [comment]
> please perform a case sensitive renaming

done

File: codegen/facelift/templates/Service.template.h [hunk 2]
```diff
@@ -171,7 +171,7 @@ class {{classExport}} {{interfaceName}} : public facelift::InterfaceBase
 
     {% endif %}
     //private:  TODO : make this field private
-    mutable QMLFrontendType* m_qmlFrontend = nullptr;
+    mutable QMLAdapterType* m_QMLAdapter = nullptr;
```
Comment (jacky309) [inline codegen/facelift/templates/Service.template.h]: please review your commit first ! Thanks

File: doc/page-reference.h [hunk 3]
```diff
@@ -34,7 +34,7 @@ The following table describes the C++ classes which are generated from the QFace
 |----------------------------------- |:-----------:|---------------------|
 |MyInterface                         | This C++ abstract class (or interface) is the direct translation of the QFace interface definition into C++. Any concrete implementation of "MyInterface" should extend somehow this class (not necessarily directly) | none |
 |MyInterfaceImplementationBase       | This C++ abstract class extends "MyInterface" and provides a more convenient base class to write an implementation of "MyInterface". | none |
-|MyInterfaceQMLFrontend              | This class wraps an instance of "MyInterface" into an object which can be exposed directly to QML. Note that the "MyInterface" class is intentionally not QML-friendly. | none |
+|MyInterfaceQMLAdapter              | This class wraps an instance of "MyInterface" into an object which can be exposed directly to QML. Note that the "MyInterface" class is intentionally not QML-friendly. | none |
```
Comment (jacky309) [inline doc/page-reference.h]: align

File: doc/page-reference.h [hunk 4]
```diff
@@ -90,8 +90,8 @@ The following table describes the QML types:
 QML component name               | Corresponding C++ class           | Description
 -------------------------------- | --------------------------------- | -----------------------
 MyInterfaceImplementationBase    | MyInterfaceImplementationBaseQML  | This type is used to enable the implementation (backend) of "MyInterface" using the QML language.
-MyInterface <i>(you name it)</i> | MyInterfaceQMLFrontend            | This class exposes the interface defined in the IDL to QML (frontend UI)
-MyInterfaceIPCProxy              | MyInterfaceIPCQMLFrontendType     | This type is an IPC proxy for the "MyInterface" type. A proxy object implements the same interface as the actual implementation object, which typically lives in another process.
+MyInterface <i>(you name it)</i> | MyInterfaceQMLAdapter            | This class exposes the interface defined in the IDL to QML (frontend UI)
```
Comment (jacky309) [inline doc/page-reference.h]: align

File: tests/manual/test1.h [hunk 5]
```diff
@@ -48,7 +48,7 @@ class InterfaceImplementation :
 
 
 class AddressBookQMLFrontend :
```
Comment (jacky309) [inline tests/manual/test1.h]: AddressBookQMLFrontend ?

**Pull request merged**